src/physics/em/RelativisticBremModel.cc

```diff
@@ -0,0 +1,264 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file RelativisticBremModel.cc
+//---------------------------------------------------------------------------//
+#include "RelativisticBremModel.hh"
+
+#include "base/Assert.hh"
+#include "base/Algorithms.hh"
+#include "base/Constants.hh"
+
+#include "base/Range.hh"
+#include "base/CollectionBuilder.hh"
+#include "physics/base/PDGNumber.hh"
+#include "physics/base/ParticleParams.hh"
+#include "physics/em/detail/RelativisticBremData.hh"
+#include "physics/em/generated/RelativisticBremInteract.hh"
+
+#include <cmath>
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct from model ID and other necessary data.
+ */
+RelativisticBremModel::RelativisticBremModel(ModelId               id,
+                                             const ParticleParams& particles,
+                                             const MaterialParams& materials,
+                                             bool                  enable_lpm)
+{
+    CELER_EXPECT(id);
+
+    HostValue host_ref;
+
+    host_ref.ids.model    = id;
+    host_ref.ids.electron = particles.find(pdg::electron());
+    host_ref.ids.positron = particles.find(pdg::positron());
+    host_ref.ids.gamma    = particles.find(pdg::gamma());
+
+    CELER_VALIDATE(host_ref.ids,
+                   << "missing IDs (required for " << this->label() << ")");
+
+    // Save particle properties
+    host_ref.electron_mass = particles.get(host_ref.ids.electron).mass();
+
+    // Set the LPM flag (true by default)
+    host_ref.enable_lpm = enable_lpm;
+
+    // Build other data (host_ref.lpm_table, host_ref.elem_data))
+    this->build_data(&host_ref, materials);
+
+    // Move to mirrored data, copying to device
+    data_ = CollectionMirror<detail::RelativisticBremData>{std::move(host_ref)};
+    CELER_ENSURE(this->data_);
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Particle types and energy ranges that this model applies to.
+ */
+auto RelativisticBremModel::applicability() const -> SetApplicability
+{
+    Applicability electron_brem;
+    electron_brem.particle = this->host_ref().ids.electron;
+    electron_brem.lower    = this->host_ref().low_energy_limit();
+    electron_brem.upper    = this->host_ref().high_energy_limit();
+
+    Applicability positron_brem = electron_brem;
+    electron_brem.particle      = this->host_ref().ids.positron;
```

Comment:
```suggestion
    positron_brem.particle      = this->host_ref().ids.positron;
```

Reply:
Nice catch for this bug.

Reply:
@amandalund Thanks! I overlooked this comment and thought that Paul made this suggestion.